src/routes/transactions/transactions.service.ts

```diff
@@ -490,12 +489,12 @@ export class TransactionsService {
   private getFirstTransactionNonce(
     page: Page<DomainMultisigTransaction>,
   ): number | null {
-    return head(page.results)?.nonce ?? null;
+    return page.results[0].nonce ?? null;
```

Comment:
Is it possible that `page.results` is `null` in this context? It seems the prior implementation was protected against that (`head(page.results)` would return `undefined`, but `page.results[0]` would throw an uncaught error) 🤔

Reply:
Good catch! I've added optional chaining in e592ccb48f96095fc675168ec1e04d189c445dbf.